exporting/README.md

```diff
@@ -190,7 +190,7 @@ Options:
    destination = [ffff:...:0001]:2003 10.11.12.1:2003
 ```
 
-   When multiple servers are defined, Netdata will try the next one when the first one fails. This allows you to
+   When multiple servers are defined, Netdata will try the next one when the previous one fails. This allows you to
    load-balance different servers: give your external database servers in different order on each Netdata.
```

Comment:
how come is this load-balancing?

Reply:
it looks like `fault tolerance` but no load-balancing for me, am i missing something?

Reply:
Actually,
> give your external database servers in different order on each Netdata

is some kind of load-balancing.
Maybe we should remove the sentence completely?

Reply:
I think the order is not load-balancing. This order is `fault tolerance` feature.

You are saying: set different destinations to achieve load-balancing. It is not load-balancing, it is just different destinations.

Reply:
I've deleted the sentence.